pandas/core/indexes/category.py

```diff
@@ -356,8 +356,19 @@ def _wrap_setop_result(self, other, result):
         return self._shallow_copy(result, name=name)
 
     def get_values(self):
-        """ return the underlying data as an ndarray """
-        return self._data.get_values()
+        """
```

Comment:
I think you can remove this one entirely as the super class calls the _internal_get_values